src/AlgebraicGeometry/Schemes/AffineSchemes/Objects/Constructors.jl

```diff
@@ -258,6 +258,8 @@ julia> standard_spec(X)
 Spec of Localization of quotient of multivariate polynomial ring at complement of prime ideal
 ```
 """
+standard_spec(X::AbsSpec{<:Any, <:MPolyLocRing}) = Spec(MPolyQuoLocRing(ambient_coordinate_ring(X), ideal(ambient_coordinate_ring(X), [zero(ambient_coordinate_ring(X))]), inverted_set(OO(X))))
+
```

Comment:
This looks like a mistake (perhaps made during merging), as the exact same method is defined in line 236. As a result we now get these warnings:
```
WARNING: Method definition standard_spec(Oscar.AbsSpec{var"#s1896", var"#s1895"} where var"#s1895"<:(Oscar.MPolyLocRing{BaseRingType, BaseRingElemType, RingType, RingElemType, MultSetType} where MultSetType<:Oscar.AbsMPolyMultSet{BaseRingType, BaseRingElemType, RingType, RingElemType} where RingElemType where RingType where BaseRingElemType where BaseRingType) where var"#s1896") in module Oscar at /home/runner/work/Oscar.jl/Oscar.jl/src/AlgebraicGeometry/Schemes/AffineSchemes/Objects/Constructors.jl:236 overwritten at /home/runner/work/Oscar.jl/Oscar.jl/src/AlgebraicGeometry/Schemes/AffineSchemes/Objects/Constructors.jl:261.
```
@HechtiDerLachs @simonbrandhorst please fix

Reply:
will do thx

Reply:
#2512